tests/foreman/api/test_template_combination.py

```diff
@@ -0,0 +1,80 @@
+# -*- coding: utf-8 -*-
+"""Tests for template combination
+
+@Requirement: TemplateCombination
+
+@CaseAutomation: Automated
+
+@CaseLevel: Acceptance
+
+@CaseComponent: API
+
+@TestType: Functional
+
+@CaseImportance: Medium
+
+@Upstream: No
+"""
+from nailgun import entities
+
+from robottelo.decorators import tier1, skip_if_bug_open
+from robottelo.test import APITestCase
+
+
+def _delete_entities(*entities):
+    for ent in entities:
+        ent.delete()
+
+
+class TestTemplateCombination(APITestCase):
+    """TemplateCombination tests"""
+
+    @classmethod
+    def setUpClass(cls):
+        super(TestTemplateCombination, cls).setUpClass()
+        cls.hostgroup = entities.HostGroup().create()
+        cls.env = entities.Environment().create()
+
+    @classmethod
+    def tearDownClass(cls):
+        super(TestTemplateCombination, cls).tearDownClass()
+        _delete_entities(cls.hostgroup, cls.env)
+
+    def setUp(self):
+        super(TestTemplateCombination, self).setUp()
+        self.template = entities.ConfigTemplate(
+            snippet=False,
+            template_combinations=[{
+                'hostgroup_id':
+                    self.hostgroup.id,
+                'environment_id': self.env.id
+            }])
+        self.template = self.template.create()
+        comb_dct = self.template.template_combinations[0]
+        params = {
+            'environment': self.env.id,
+            'config_template': self.template.id,
+            'id': comb_dct['id'],
+            'hostgroup': self.hostgroup.id
+        }
+        self.template_combination = entities.TemplateCombination(**params)
+
+    def tearDown(self):
+        super(TestTemplateCombination, self).tearDown()
+        _delete_entities(self.template_combination, self.template)
+
+    @tier1
+    @skip_if_bug_open('bugzilla', 1369737)
```

Comment:
I can't remove because this is not working on sat < 6.3, so it would generate false positives there. Check: #4063